mep/common/forms.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Mapping
+
+from django import forms
+from django.utils.text import mark_safe
+
+class CheckboxFieldset(forms.CheckboxSelectMultiple):
+
```

Comment:
No docstring

Reply:
Thank you. I thought I'd added it, but I believe I shuffled it away when merging changes.